gradle/dependencies.gradle

```diff
@@ -70,13 +70,13 @@ ext {
             mapboxServices           : "com.mapbox.mapboxsdk:mapbox-sdk-services:${version.mapboxServices}",
 
             // Mapbox plugins
-            mapboxPluginBuilding     : "com.mapbox.mapboxsdk:mapbox-android-plugin-building-v7:${version.mapboxPluginBuilding}",
-            mapboxPluginPlaces       : "com.mapbox.mapboxsdk:mapbox-android-plugin-places-v7:${version.mapboxPluginPlaces}",
-            mapboxPluginLocalization : "com.mapbox.mapboxsdk:mapbox-android-plugin-localization-v7:${version.mapboxPluginLocalization}",
-            mapboxPluginTraffic      : "com.mapbox.mapboxsdk:mapbox-android-plugin-traffic-v7:${version.mapboxPluginTraffic}",
+            mapboxPluginBuilding     : "com.mapbox.mapboxsdk:mapbox-android-plugin-building-v8:${version.mapboxPluginBuilding}",
```

Comment:
maybe we should extract the v8 major version into a separate constant?

Reply:
Done

![Screen Shot 2019-06-12 at 9 33 45 AM](https://user-images.githubusercontent.com/4394910/59369392-31175680-8cf5-11e9-8e81-52274c6adc3d.png)
![Screen Shot 2019-06-12 at 9 33 42 AM](https://user-images.githubusercontent.com/4394910/59369393-31175680-8cf5-11e9-9e4e-c3b7ae6b6dd1.png)